lib/internal/Magento/Framework/Mview/View/Changelog.php

```diff
@@ -95,10 +112,46 @@ public function create()
                 ['unsigned' => true, 'nullable' => false, 'default' => '0'],
                 'Entity ID'
             );
+
+            foreach ($this->initAdditionalColumnData() as $columnData) {
```

Comment:
Can we use JSON filed here?

Reply:
Can you describe in more details what you mean?

Reply:
I don`t like columns as JSON. As we want to have more clear structure with strict types